server/app/services/program/ProgramServiceImpl.java

```diff
@@ -207,6 +207,7 @@ public ErrorAnd<ProgramDefinition, CiviFormError> updateProgramDefinition(
             .build()
             .toProgram();
     return ErrorAnd.of(
+        // Note: This method seems heavy-handed for non question updates.
```

Comment:
Nit: I'd consider removing this comment since there's nothing actionable from it. If this is something we should fix or an idea for a future improvement, we should either 1) file an issue + TODO or 2) expand the comment to include a potential solution or time when we should fix.

Reply:
Yeah... There's so much stuff like this given the rapid development that a TODO for all of it feels like too much too :D

I removed the usage comments and kept the method one.